src/browser/client-bridge/index.js

```diff
@@ -1,46 +1,25 @@
 "use strict";
 
 const path = require("path");
-const Promise = require("bluebird");
-const browserify = require("browserify");
+const fs = require("fs");
 const ClientBridge = require("./client-bridge");
 
+const bundlesCache = {};
+
 exports.ClientBridge = ClientBridge;
 
-exports.build = (browser, opts = {}) => {
-    const script = browserify({
-        entries: "./index",
-        basedir: path.join(__dirname, "..", "client-scripts"),
-    });
-
-    script.transform(
-        {
-            sourcemap: false,
-            global: true,
-            compress: { screw_ie8: false }, // eslint-disable-line camelcase
-            mangle: { screw_ie8: false }, // eslint-disable-line camelcase
-            output: { screw_ie8: false }, // eslint-disable-line camelcase
-        },
-        "uglifyify",
-    );
-
-    const lib = opts.calibration && opts.calibration.needsCompatLib ? "./lib.compat.js" : "./lib.native.js";
-    const ignoreAreas = opts.supportDeprecated ? "./ignore-areas.deprecated.js" : "./ignore-areas.js";
-
-    script.transform(
-        {
-            aliases: {
-                "./lib": { relative: lib },
-                "./ignore-areas": { relative: ignoreAreas },
-            },
-            verbose: false,
-        },
-        "aliasify",
-    );
-
-    return Promise.fromCallback(cb => script.bundle(cb)).then(buf => {
-        const scripts = buf.toString();
-
-        return ClientBridge.create(browser, scripts);
-    });
+exports.build = async (browser, opts = {}) => {
```

Comment:
I wanted to rewrite this file to typescript, but a lot of testing files depend on ability to mock this "build" method and we won't be able to do that with esm exports